lms/djangoapps/certificates/tests/test_api.py

```diff
@@ -92,7 +93,8 @@ def setUp(self):
         self.course = CourseFactory.create(
             org='edx',
             number='verified',
-            display_name='Verified Course'
+            display_name='Verified Course',
+            end=datetime.datetime.now()
```

Comment:
It would be good to have a test here that expects `is_dateunavailable` to be `True`.